test/sim/moves/metalburst.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const assert = require('./../../assert');
+const common = require('./../../common');
+
+let battle;
+
+describe('Metal Burst', function () {
+	afterEach(function () {
+		battle.destroy();
+	});
+
+	it('should run conditions for submove', function () {
+		battle = common.createBattle({seed: [1, 2, 3, 4]}); // Snorlax needs to sleep for more than 1 turn
```

Comment:
Swap this to doubles so it doesn't need an RNG seed.

Reply:
Complete.